conda_smithy/cli.py

```diff
@@ -76,18 +92,19 @@ def __init__(self, parser):
         scp = self.subcommand_parser
         scp.add_argument("recipe_directory", help="The path to the source recipe directory.")
         scp.add_argument("--feedstock-directory", default='./{package.name}-feedstock',
-                        help="Target directory, where the new feedstock git repository should be "
-                             "created. (Default: './<packagename>-feedstock')")
-        scp.add_argument("--no-git-repo", action='store_true',
-                                       default=False,
-                                       help="Do not init the feedstock as a git repository.")
+                         help="Target directory, where the new feedstock git repository should be "
+                         "created. (Default: './<packagename>-feedstock')")
+        scp.add_argument("--no-git-repo", action='store_true', default=False,
+                         help="Do not init the feedstock as a git repository.")
```

Comment:
Not exactly related, but I think we should axe this. Raised an issue about dropping it some time back and got nothing but crickets. Seems silly to spend any more time supporting something that is unused.

Reply:
xref: https://github.com/conda-forge/conda-smithy/issues/333